### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by RR-user**

### CODE REVIEW

File: JupyterNotebook_Sessao11-Questao4.ipynb [hunk 1]
```diff
@@ -15,7 +15,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Questão 4\n",
+    "# Questão 4 blah blah +1\n",
```
Comment (miguelbalmeida) [inline JupyterNotebook_Sessao11-Questao4.ipynb]: I don't understand this part, what does "blah" mean?

**Review by miguelbalmeida — CHANGES_REQUESTED** [review]
I do not approve.

File: JupyterNotebook_Sessao11-Questao4.ipynb (hunk 1 above)
Comment (aldo-arevalo) [reply JupyterNotebook_Sessao11-Questao4.ipynb]: I agree.

**Review by aldo-arevalo — CHANGES_REQUESTED** [review]
It doesn't make sense. Check again.

**Pull request merged**